### PR TITLE
Clean up feature macro table

### PR DESCRIPTION
**Pull request by rsandifo-arm**
---
name: Pull request
about: Technical issues, document format problems, bugs in scripts or feature proposal.

---

**Thank you for submitting a pull request!**

If this PR is about a bugfix:

Please use the bugfix label and make sure to go through the checklist below.

If this PR is about a proposal:

We are looking forward to evaluate your proposal, and if possible to
make it part of the Arm C Language Extension (ACLE) specifications.

We would like to encourage you reading through the [contribution
guidelines](../CONTRIBUTING.md), in particular the section on [submitting
a proposal](../CONTRIBUTING.md#proposals-for-new-content).

Please use the proposal label.

As for any pull request, please make sure to go through the below
checklist.

Checklist: (mark with ``X`` those which apply)

* [X] If an issue reporting the bug exists, I have mentioned it in the
      PR (do not bother creating the issue if all you want to do is
      fixing the bug yourself).
* [X] I have added/updated the `SPDX-FileCopyrightText` lines on top
      of any file I have edited. Format is `SPDX-FileCopyrightText:
      Copyright {year} {entity or name} <{contact informations}>`
      (Please update existing copyright lines if applicable. You can
      specify year ranges with hyphen , as in `2017-2019`, and use
      commas to separate gaps, as in `2018-2020, 2022`).
* [X] I have udated the `Copyright` section of the sources of the
      specification I have edited (this will show up in the text
      rendered in the PDF and other output format supported). The
      format is the same described in the previous item.
* [X] I have run the CI scripts (if applicable, as they might be
      tricky to set up on non-*nix machines). The sequence can be
      found in the [contribution
      guidelines](../CONTRIBUTING.md#continuous-integration). Don't
      worry if you cannot run these scripts on your machine, your
      patch will be automatically checked in the Actions of the pull
      request.
* [X] The pull request is done against the branch `next-release`.

### CODE REVIEW

File: main/acle.md [hunk 1]
```diff
@@ -1123,6 +1127,8 @@ available. This feature also implies support for the Q flag.
 ACLE 2.0 for A-profile. They are fully supported for M and R-profiles.
 This macro is defined for AArch32 only.
 
+<span id="32-bit-simd-instructions"></span>
```
Comment (fpetrogalli) [inline main/acle.md:1130]: This might be a bit of a nuisance, but in other places where we have added this `span` mechanism I have added a comment to make sure it doesn't get accidentally removed. May I ask you to do the same?

File: main/acle.md [hunk 2]
```diff
@@ -1406,8 +1417,9 @@ this implies:
 
 ### Javascript floating-point conversion
 
-`__ARM_FEATURE_JCVT` is defined to 1 if the FJCVTZS (AArch64) or
-VJCVT (AArch32) instruction and the associated intrinsic is available.
+`__ARM_FEATURE_JCVT` is defined to 1 if the FJCVTZS (AArch64)
+or VJCVT (AArch32) instruction and the [associated
+intrinsic]#floating-point-data-processing-intrinsics is available.
```
Comment (fpetrogalli) [inline main/acle.md:1422]: I dare grammar :)

```suggestion
intrinsic]#floating-point-data-processing-intrinsics are available.
```

File: main/acle.md [hunk 3]
```diff
@@ -1627,73 +1639,73 @@ be found in [[BA]](#BA).
...
-| `__ARM_SIZEOF_WCHAR_T` [[ssec-Imptype]](#implementation-defined-type-properties)                                                                                     | Size of `wchar_t`: 2 or 4                                                                          | 2           |
-| `__ARM_WMMX` [[ssec-WMMX]](#wireless-mmx)                                                                                                                            | Wireless MMX extension (32-bit-only)                                                               | 1           |
-| `__ARM_FEATURE_MOPS` [[ssec-MOPS]](#memcpy-family-of-memory-operations-standarization-instructions---mops)                                                           | memcpy, memset, and memmove family of operations standardization instructions                      | 1           |
+| **Macro name**                                                                                                                                          | **Meaning**                                                                                        | **Example** |
```
Comment (fpetrogalli) [inline main/acle.md:1642]: The changes in this table are hard to review (probably) because of the sorting. May I ask you to revert the sorting and submit it in a separate PR?

**Review by fpetrogalli — CHANGES_REQUESTED** [review]
Thank you @rsandifo-arm !! This looks great, you saved me some work! I just have some minor comments.

Francesco

File: main/acle.md (hunk 3 above)
Comment (fpetrogalli) [reply main/acle.md:1642]: Never mind - I looked at the individual commit and LGTM, no need to migrate it to a separate PR.

File: main/acle.md (hunk 2 above)
Comment (rsandifo-arm) [reply main/acle.md:1422]: Actually, this is missing (…) around the link, oops.  The text is “associated intrinsic”, so the singular itself is right.

Will fix.

Comment (fpetrogalli) [reply main/acle.md:1422]: OK, but I thought that the verb referred to both the instruction *and* the associated intrinsic. That's two things to me :)

Comment (rsandifo-arm) [reply main/acle.md:1422]: Fair :-)  I'll fix that while I'm here, thanks.

**Review by fpetrogalli — APPROVED** [review]
Thank you!
